internal/routes/server.go

```diff
@@ -409,6 +409,8 @@ func statsRoutes(r *mux.Router, c *stats.Controller) {
 
 	r.Handle("/realm/external-issuers.csv", c.HandleRealmExternalIssuersStats(stats.StatsTypeCSV)).Methods("GET")
 	r.Handle("/realm/external-issuers.json", c.HandleRealmExternalIssuersStats(stats.StatsTypeJSON)).Methods("GET")
+
+	r.Handle("/realm/key-server-stats.json", c.HandleKeyServerStats(stats.StatsTypeJSON)).Methods("GET")
```

Comment:
This is already under the /stats parent route, so I think we can drop `-stats` from the path

Reply:
broader point - should this be more under the realm stats that already exist?

Reply:
See my review comment ;)

Reply:
I think we'd just need to align dates for the Day field. It looks like the existing stats use timeutils.Midnight rather than UTCMidnight?

Reply:
Just kidding `timeutils.Midnight(time.Now().UTC())` is the same thing

Reply:
I've added in caching. At the moment I have them separate because they're stored in a different table so we'd need to re-do the json marshal to merge the two gorm structs to a single output struct. I figured the other stats were split out so this would be the easiest thing for now.

I also don't have a good solution for CSVing this because the distributions add an array dimension that's easy to do in json, but not CSV.

Reply:
I’d suggest joining them with a pipe character

Reply:
That works. esp since they're fixed-length. I might make that the next PR - this seems like a good place to stop.